src/components/Attendance/AttendanceTracker/AttendanceTracker.js

```diff
@@ -8,19 +8,26 @@ export default class AttendanceTracker extends Component {
   constructor(props) {
     super(props);
 
-    this.state = {};
+    this.state = {
+      chosenCohort: 0
+    };
   }
 
   render() {
-    console.log(this.state);
-    return this.props.students.map(student => (
-      <div key={student.dmId} className="attendance-tracker-container">
-        <AttendanceButtonInput student={student} />
-      </div>
-    ));
+    return (
+      this.props.students &&
+      this.props.students[
+        this.props.cohort || this.state.chosenCohort
```

Comment:
In Attendance.js, just pass down students.classSession that you want, then you should be able to just map the array that's passed in, rather than first having to check if students exists, then locating the right array.